data_model/src/block.rs

```diff
@@ -95,10 +95,10 @@ mod committed {
         pub struct CommittedBlock {
             /// Block header
             pub header: BlockHeader,
-            /// Array of rejected transactions.
-            pub rejected_transactions: Vec<VersionedRejectedTransaction>,
             /// array of transactions, which successfully passed validation and consensus step.
             pub transactions: Vec<VersionedValidTransaction>,
+            /// Array of rejected transactions.
```

Comment:
Is this change necessary? IMO it changes the schema for no apparent reason. 

Should we just agree that fields should be sorted alphabetically, to prevent unnecessary changes in the schema?

Reply:
ok, reverting this. It's not relevant

Reply:
> Should we just agree that fields should be sorted alphabetically, to prevent unnecessary changes in the schema?

I don't support this idea

Reply:
> Is this change necessary? IMO it changes the schema for no apparent reason.

reason was to make it more in line with order of fields in the `BlockHeader`

Reply:
I can revert it, but considering recent schema changes I thought it might not be problematic

Reply:
We should not be changing the order of fields in the schema willy nilly. So either we impose an order and generate the schema exactly as seen in the struct, or we decouple the order of fields in the schema from the order of field definitions in the struct. 

Refactors should never break the API.